### DISCUSSION TITLE
No Signal Issue

### DISCUSSION THREAD

**Discussion by mdb17** [awawa-dev/HyperHDR#322] [Q&A] [answered]
I am hoping this will be a easy fix. I had to swap out my capture card and now whenever I don't have a signal I get the rainbow pattern in the image. I have checked and I don't have an idle effect selected. Any suggestions on what I am doing wrong?
![1B5FA750-5D58-4CCB-A0B3-4224A9376C45](https://user-images.githubusercontent.com/20051857/185801070-7b475ccf-507a-4d50-96bd-47c9b127808c.jpeg)

**Comment by awawa-dev** [comment, marked as answer]
Hi
This isn't HyperHDR effect but your grabber is just sending such no-signal rainbow board. Read here how to make HyperHDR recognize this board and turn off the LED strip:  https://www.hyperhdr.eu/2021/08/hyperhdr-v17beta-preview.html (point nr 4 - the automatic signal detection)

**Reply by mdb17** [reply]
see I knew it had to be something easy
